tests/setup_test.go

```diff
@@ -2401,3 +2403,237 @@ func TestUpdateOrganization(t *testing.T) {
+	expectedErr = "remote error internal"
+	if err == nil {
+		t.Fatalf("expected error %v, got nil err", expectedErr)
+	}
+	if err.Error() != expectedErr {
+		t.Fatalf("expected err %v, got err: %v", expectedErr, err)
+	}
+
+	//test user02 accept invitation
+	_, _, err = gwClientUser01.CreateOrgInvitation(ctx, "org01", &gwapitypes.CreateOrgInvitationRequest{UserRef: "user02", Role: cstypes.MemberRoleMember})
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+
+	_, err = gwClientUser02.OrgInvitationAction(ctx, "org01", "user02", &gwapitypes.OrgInvitationActionRequest{Action: gwapitypes.Accept})
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+
+	_, _, err = gwClientUser02.GetOrgInvitation(ctx, "org01", "user02")
+	expectedErr = "remote error internal"
+	if err == nil {
+		t.Fatalf("expected error %v, got nil err", expectedErr)
+	}
+	if err.Error() != expectedErr {
+		t.Fatalf("expected err %v, got err: %v", expectedErr, err)
+	}
+
+	org01Members, _, err := gwClientUser01.GetOrgMembers(ctx, "org01")
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+	if len(org01Members.Members) != 2 {
+		t.Fatalf("expected 2 members got: %d", len(org01Members.Members))
+	}
+
+	//create invitation org not exists
+	_, _, err = gwClientUser01.CreateOrgInvitation(ctx, "org02", &gwapitypes.CreateOrgInvitationRequest{UserRef: "user02", Role: cstypes.MemberRoleMember})
+	expectedErr = "remote error notexist"
+	if err == nil {
+		t.Fatalf("expected error %v, got nil err", expectedErr)
+	}
+	if err.Error() != expectedErr {
+		t.Fatalf("expected err %v, got err: %v", expectedErr, err)
+	}
+
+	//create invitation user just org member
+	_, _, err = gwClientUser01.CreateOrgInvitation(ctx, "org01", &gwapitypes.CreateOrgInvitationRequest{UserRef: "user02", Role: cstypes.MemberRoleMember})
+	expectedErr = "remote error forbidden"
+	if err == nil {
+		t.Fatalf("expected error %v, got nil err", expectedErr)
+	}
+	if err.Error() != expectedErr {
+		t.Fatalf("expected err %v, got err: %v", expectedErr, err)
+	}
+
+	//create invitation user not exists
+	_, _, err = gwClientUser01.CreateOrgInvitation(ctx, "org01", &gwapitypes.CreateOrgInvitationRequest{UserRef: "user03", Role: cstypes.MemberRoleMember})
+	expectedErr = "remote error notexist"
+	if err == nil {
+		t.Fatalf("expected error %v, got nil err", expectedErr)
+	}
+	if err.Error() != expectedErr {
+		t.Fatalf("expected err %v, got err: %v", expectedErr, err)
+	}
+
+	//test create org invitation and accept after invitations disabled
+	_, _, err = gwClientUser01.CreateOrg(ctx, &gwapitypes.CreateOrgRequest{Name: "org02", Visibility: gwapitypes.VisibilityPublic})
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+	_, _, err = gwClientUser01.CreateOrgInvitation(ctx, "org02", &gwapitypes.CreateOrgInvitationRequest{UserRef: "user02", Role: cstypes.MemberRoleMember})
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+
```

Comment:
Please add a check where an admin user directly adds a user to an org with an existing invitation and check that the invitation is removed.